src/python/grpcio/grpc/_cython/_cygrpc/grpc.pxi

```diff
@@ -28,15 +28,20 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 cimport libc.time
-from libc.stdint cimport int64_t, uint32_t, int32_t
 
 
-cdef extern from "grpc/support/alloc.h":
+cdef extern from "grpc/_cython/loader.h":
+
+  ctypedef int int32_t
+  ctypedef unsigned uint32_t
+  ctypedef long int64_t
```

Comment:
It's a Cython thing in the sense that the type doesn't need to be the exact type, it just needs to be a type supporting the same interface as the type actually being used that Cython is aware of elsewhere (so the `ctypedef` keyword is a kind of misnomer in this scenario).